src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LbfgsPredictorBase.cs

```diff
@@ -71,7 +71,7 @@ public abstract class OptionsBase : LearnerInputBaseWithWeight
             [Argument(ArgumentType.LastOccurenceWins, HelpText = "Init weights diameter", ShortName = "initwts", SortOrder = 140)]
             [TGUI(Label = "Initial Weights Scale", SuggestedSweeps = "0,0.1,0.5,1")]
             [TlcModule.SweepableFloatParamAttribute("InitWtsDiameter", 0.0f, 1.0f, numSteps: 5)]
-            public float InitWtsDiameter = 0;
+            public float InitialWeightsDiameter = 0;
```

Comment:
>InitialWeightsDiameter [](start = 25, length = 22)

So when you do this, you need to take old name and put it in ShortName in ArgumentAttribute like this ShortName="initwts,initwtsDiameter"

Reply:
1. does it apply to all the other renamings as well ?  .
2. why  we adding it to ShortName like that ?  

---
In reply to: [260893400](https://github.com/dotnet/machinelearning/pull/2761#discussion_r260893400) [](ancestors = 260893400)

Reply:
1. Yes
2. for cmd back compatibility.

---
In reply to: [260894964](https://github.com/dotnet/machinelearning/pull/2761#discussion_r260894964) [](ancestors = 260894964,260893400)